From 8e3eabba84f92f21760bfc8cb8070f6f18f15637 Mon Sep 17 00:00:00 2001
From: Philipp Kerling <yol@casix.org>
Date: Wed, 15 Apr 2020 20:07:26 +0200
Subject: [PATCH] Disable VAAPI/VDPAU MPEG-2 hwaccel by default

radeon mesa driver still seems to have issues with MPEG-2 and with
current and even older hardware hardware-decoding MPEG-2 does not give
substantial benefit.

Fixes #17651 (kind of)

See also mpv-player/mpv@3dd59db
---
 system/settings/linux.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/system/settings/linux.xml b/system/settings/linux.xml
index a500ce9fe27e..801be4848c45 100644
--- a/system/settings/linux.xml
+++ b/system/settings/linux.xml
@@ -26,7 +26,7 @@
           <requirement>HAVE_LIBVDPAU</requirement>
           <visible>false</visible>
           <level>3</level>
-          <default>true</default>
+          <default>false</default>
           <dependencies>
             <dependency type="enable">
               <condition setting="videoplayer.usevdpau" operator="is">true</condition> <!-- USE VDPAU -->
@@ -74,7 +74,7 @@
             </dependency>
           </dependencies>
           <level>3</level>
-          <default>true</default>
+          <default>false</default>
           <control type="toggle" />
         </setting>
         <setting id="videoplayer.usevaapimpeg4" type="boolean" parent="videoplayer.usevaapi" label="13449" help="13450">
